web/src/js/index.js

```diff
@@ -231,43 +235,55 @@ function getExecutionLog(executionLogUrl) {
   })
     .then((response) => response.json())
     .then((data) => {
-      app.executionLogTable = data["response"]["body"];
       app.executionLogKey = decodeURIComponent(executionLogUrl);
+      app.executionLogTable = data["response"]["body"];
 
-      setTimeout(function () {
-        app.executionLogDataTables = $("#execution-log-table").DataTable({
-          autoWidth: true,
-          columnDefs: [
-            { orderable: false, targets: [6] },
-            { className: "dt-center", targets: [6] },
-          ],
-          dom: "lrti",
-          paging: false,
-        });
-        app.showExecutionLogLoadingGif = false;
-      }, 10);
-
-      // Get execution mode
-
-      if (data["response"]["body"][0][7] == "True") {
-        app.executionLogMode = "Dry Run";
-      } else {
-        app.executionLogMode = "Destroy";
+      if (data["response"]["is_compressed"]) {
+        try {
+          let compressedData = Uint8Array.from(
+            atob(app.executionLogTable),
+            (c) => c.charCodeAt(0)
+          );

```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/servian/aws-auto-cleanup/pullRequest?prid=9792603)
```suggestion
        );
```